DelvUI/Plugin.cs

```diff
@@ -255,13 +255,14 @@ protected virtual void Dispose(bool disposing)
             UiBuilder.RebuildFonts();
 
             ChatHelper.Instance.Dispose();
-            ConfigurationManager.Instance.Dispose();
             FontsManager.Instance.Dispose();
             GlobalColors.Instance.Dispose();
             MouseOverHelper.Instance.Dispose();
             PartyManager.Instance.Dispose();
             TexturesCache.Instance.Dispose();
             TooltipsHelper.Instance.Dispose();
+
+            ConfigurationManager.Instance.Dispose();

```

Comment:
```suggestion
            
            // This needs to remain last to avoid race conditions
            ConfigurationManager.Instance.Dispose();
```
IMO this is a good example of a place we actually want a comment